### PR TITLE
[ID] Add Glossary index and missing glossaries

### PR DESCRIPTION
**Pull request by daud-ahn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Glossary/Hyperdash/id.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+stub: true
+---
+
+# Hyperdash
+
+**Hyperdash** adalah istilah yang digunakan di [osu!catch](/wiki/Game_mode/osu!catch) untuk gerakan yang dilakukan saat [hyperfruit](/wiki/Hit_object/Hyperfruit) dihasilkan. Saat *modding* sebuah beatmap, orang-orang cenderung menautkan hyperfruit dan target [objek aktif](/wiki/Glossary/Active_object) untuk mendefinisikan gerakan hyperdash (misalnya `00:01:00 (1,2) -` di mana 1 adalah hyperfruit, dan 2 objek aktif target).
```
Comment (ZeroPyrozen) [inline wiki/Glossary/Hyperdash/id.md:7]:
```suggestion
**Hyperdash** adalah istilah yang digunakan di [osu!catch](/wiki/Game_mode/osu!catch) untuk gerakan yang dilakukan saat [hyperfruit](/wiki/Hit_object/Hyperfruit) dihasilkan. Saat *modding* sebuah beatmap, orang-orang cenderung menautkan hyperfruit dan target [objek aktif](/wiki/Glossary/Active_object) untuk mendefinisikan gerakan hyperdash (misalnya `00:01:00 (1,2) -` di mana 1 adalah hyperfruit dan 2 objek aktif target).
```

File: wiki/Glossary/Snapping/id.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Snapping
+
+Di [osu!catch](/wiki/Game_mode/osu!catch) setiap [dash](/wiki/Glossary/Dash) dan [hyperdash](/wiki/Glossary/Hyperdash) dapat diklasifikasikan menjadi dua snapping yang berbeda, *basic-snapped* atau *higher-snapped*.
```
Comment (ZeroPyrozen) [inline wiki/Glossary/Snapping/id.md:3]:
```suggestion
Di [osu!catch](/wiki/Game_mode/osu!catch) setiap [dash](/wiki/Glossary/Dash) dan [hyperdash](/wiki/Glossary/Hyperdash) dapat diklasifikasikan menjadi dua snapping yang berbeda yaitu, *basic-snapped* atau *higher-snapped*.
```

File: wiki/Glossary/Snapping/id.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Istilah **basic-snapped** digunakan untuk semua [dash](/wiki/Glossary/Dash) atau [hyperdash](/wiki/Glossary/Hyperdash) yang waktu antara [objek aktif](/wiki/Glossary/Active_object) setidaknya dua kali lebih lama dari waktu yang diperlukan untuk mengizinkan dash atau hyperdash. Misalnya, hyperdash antara objek yang terpisah selama 250 ms di Platter diklasifikasikan sebagai hyperdash basic-snapped.
+
+## Higher-snapped
+
+Istilah **higher-snapped** digunakan untuk semua [dash](/wiki/Glossary/Dash) atau [hyperdash](/wiki/Glossary/Hyperdash) yang tidak memenuhi persyaratan untuk menjadi basic-snapped (yaitu waktu antara objek lebih pendek dari ambang batas untuk diklasifikasikan sebagai basic).
+
+## Tabel referensi snapping
+
+| Kesulitan | Dash basic-snapped | Dash higher-snapped | Hyperdash basic-snapped | Hyperdash higher-snapped |
```
Comment (ZeroPyrozen) [inline wiki/Glossary/Snapping/id.md:15]:
```suggestion
| Tingkat kesulitan | Dash basic-snapped | Dash higher-snapped | Hyperdash basic-snapped | Hyperdash higher-snapped |
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Review by ZeroPyrozen — APPROVED** [review]
All good 👍

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**